SECURITY.md

```diff
@@ -0,0 +1,8 @@
+# Vulnerability Reporting Policy
+
+If you think or suspect that you have discovered a new security vulnerability in this project, please __do not__ disclose it on GitHub, e.g. in an issue, a PR, or a discussion. Any such disclosure will be removed/deleted on sight, to promote orderly disclosure, as per the [Eclipse Foundation Security Policy][policy].
```

Comment:
`Eclipse Foundation Vulnerability Reporting Policy` instead of `Eclipse Foundation Security Policy`

Reply:
I missed that you wanted to change the name of the policy. I will change that.